src/models/baseline.py

```diff
@@ -1,12 +1,51 @@
-from models.base import TorchModel
+from .base import TorchModel
 
-from utils import get_activation_function
-from utils import DEVICE
+from ..utils import get_activation_function
+from ..utils import DEVICE
 
 import torch
 import torch.nn as nn
 
 
+class TransformerEncoderLayer(nn.Module):
```

Comment:
Why you decided to write implementation of `TransformerEncoderLayer` here? Can you explain please?

Reply:
Resolved in DM